components/avatar/avatar.mdx

```diff
@@ -15,22 +15,10 @@ import * as AvatarStories from './avatar.stories.js';
   <Story of={AvatarStories.Default} />
 </Canvas>
 
-## Icon
+## Variants
```

Comment:
Could we update the doc to explain all these new behaviours and how it will works?

Reply:
All of that is self-explained in the properties description, I don't think it's necessary to add more documentation here. what do you think @braddialpad?

Reply:
IMO It would be cool to explain which behaviours bring Avatar beyond the props explanation.

This facilitates the quick understanding of what avatar performs.

```
For example explain the herarchy of what Avatar will show.

The Avatar component is designed to prioritize different sources for content display.
It will sequentially check for the availability of an image source (imageSrc), an icon name (iconName), and finally, the full name (fullName) to create initials. 

For example `Jaqueline Nackos` will be `JN`, some edge cases here are:
`Jaqueleine` = ??
`Jaqueline Nackos Lorem Ipsum` = ??

In the absence of any of these sources, the component will throw an error, as the fullName prop is required.

```